<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFormat</class>
 <widget class="QDialog" name="DlgFormat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>615</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>615</width>
    <height>490</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Format Log Files</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgFormat{
background-color: #cccccc;
}

QLabel{
color: blue;
}

QLineEdit{
background-color: #DDF1FF;
color: #0076C8;
padding-left: 10px;
}

QProgressBar:horizontal {
border: 1px solid gray;
border-radius: 3px;
background: white;
padding: 1px;
text-align: right;
margin-right: 4ex;
}
QProgressBar::chunk:horizontal {
background: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5, stop: 0 green, stop: 1 white);
margin-right: 2px; /* space */
width: 10px;
}

QPushButton{
color: #ffffff;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5ddced, stop: 0.1 #59bfec, stop: 0.5 #5494eb, stop: 0.9 #517eea, stop: 1 #5077ea);
border-radius: 5;
font-weight:bold;
}
QPushButton:pressed{
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5077ea, stop: 0.1 #517eea, stop: 0.5 #5494eb, stop: 0.9 #59bfec, stop: 1 #5ddced);
}
QPushButton:disabled{
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 1 #a7a7a7, stop: 0 #f4f4f4);
color:#9d9d9d;
}


QListView{
background-color: #DDF1FF;
alternate-background-color: #A1A1A1;
font-family: &quot;calibre&quot;;
font-style: normal;
font-size: 13px;
color: black;
}
QListView::item{
padding-top: 4px;
padding-bottom: 4px;
}
QListView::indicator{
width: 40px;
height: 18px;
}

/* !! Custom Scroll Bar !! */
QScrollBar:vertical {
background: #ededed; width: 8px; 
}
QScrollBar::handle:vertical {
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5077ea,  stop: 1 #5ddced); min-height: 40px;
}
QScrollBar::handle:vertical:hover {
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 1 #0037db,  stop: 0 #0037db);
}
QScrollBar::add-line:vertical { 
background: #75a1ff; height: 0px; subcontrol-position: bottom;   subcontrol-origin: margin; 
}
QScrollBar::sub-line:vertical { 
background: #75a1ff; height: 0px; subcontrol-position: top;  subcontrol-origin: margin;
}

QScrollBar:horizontal {
background: #ededed; 
height:8px;
}
QScrollBar::handle:horizontal {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #5077ea,  stop: 1 #5ddced);
min-width: 40px;
}
QScrollBar::handle:horizontal:hover {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 1 #0037db,  stop: 0 #0037db);
}
QScrollBar::add-line:horizontal {
background: #75a1ff;
width: 0px;
subcontrol-position: bottom; 
subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
background: #75a1ff;
width: 0px;
subcontrol-position: top;
subcontrol-origin: margin;
}
</string>
  </property>
  <widget class="QListView" name="lstFileNames">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>525</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="prgStatus">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>474</y>
     <width>600</width>
     <height>15</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFormatString">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>287</y>
     <width>466</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSaveFileTo">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>340</y>
     <width>513</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtnBrowseSaveFile">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>339</y>
     <width>78</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeadFormat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>590</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter comma seperated strings to look for in log files. [Do not add space !]</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeadFormat_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>324</y>
     <width>590</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter file path to save formatted log file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtnFormat">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>421</y>
     <width>107</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string>Format</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtnCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>423</y>
     <width>107</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtn_AddFiles">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>59</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkbxCaseSensitivity">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>285</y>
     <width>130</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>Case Sensitive</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtn_ClearFiles">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPrefixFileWith">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>377</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeadFormat_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>386</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter a string to prefix file names with [optional]...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
